frontend/src/app/filter/range-filter.component.html

```diff
@@ -1,5 +1,5 @@
-<div class="range">
-    <span>{{data.min}} - {{data.max}}</span>
-    <p-slider [animate]="true" [ngModel]="getDisplayData(data)"
-        [range]="true" [min]="min" [max]="max" (onSlideEnd)="update(getFilterData($event))"></p-slider>
+<div class="range" *ngIf="data$ | async as data">
+    <span>{{data[0]}} - {{data[1]}}</span>
```

Comment:
Is it a lot of work to convert this to `data.min` and `data.max`?
I understand that for two values the confusion is minor, but for readability it helps to have properties with some semantic information.